### PR TITLE
[WIP] Child care policy

### PR DESCRIPTION
**Pull request by skade**
Uses our old child-care mechanism, extracted from a blog-post

### CODE REVIEW

File: parents.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 ---
 layout: blue_page
 permalink: /parents/
-title: Parents
+title: Little Rustaceans
 ---
 
-RustFest provides childcare.
+Like in the sea the older ones should care for the little ones. With that in mind, we provide child care for our conference attendees. We cater to any needs, but need to know a bit about about your kid!
```
Comment (badboy) [inline parents.md]:
```patch
- about about
+ about
```

File: parents.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 
-Info is coming _real soon_, watch this space or our [Twitter feed](https://twitter.com/rustfest).
+If you would like to bring your own rustacean, [contact us](mailto:team@rustfest.eu?subject=Child%20care%20in%20Rome) before 2018-10-25 so we can plan for child care. Requests after that time cannot be guaranteed, but still get in touch. Please include:
```
Comment (badboy) [inline parents.md]: I know this writing is intendend to be on the fun side, but should we make it more clear that by "your own rustacean" we mean kids?

Comment (skade) [reply parents.md]: Ha. Well, this is already the toned down version from last times... I'll rephrase that.

**Comment by skade** [comment]
Fixed comments and rebased.

**Pull request merged**